### PR TITLE
Add hook docs: lib/infrastructure/class-wp-rest-server.php

### PR DESCRIPTION
**Pull request by adamsilverstein**
Note, see: https://github.com/WP-API/WP-API/issues/1549

* Add missing hook docs
* Some language and punctuation cleanup

### CODE REVIEW

File: lib/infrastructure/class-wp-rest-server.php [hunk 1]
```diff
@@ -251,9 +252,18 @@ public function serve_request( $path = null ) {
 		// http://miki.it/blog/2014/7/8/abusing-jsonp-with-rosetta-flash/
 		$this->send_header( 'X-Content-Type-Options', 'nosniff' );
 
-		// Proper filter for turning off the JSON API. It is on by default.
+		/**
+		 * Filter whether the JSON API is enabled.
```
Comment (rachelbaker) [inline lib/infrastructure/class-wp-rest-server.php]: @adamsilverstein JSON API -> WP REST API

Comment (adamsilverstein) [inline lib/infrastructure/class-wp-rest-server.php]: Right! Will fix.

**Comment by rachelbaker** [comment]
@adamsilverstein other than the one minor text change this is :+1:

**Comment by adamsilverstein** [comment]
will fix, having fun!


On Tue, Sep 15, 2015 at 6:46 PM, Rachel Baker <notifications@github.com>
wrote:

> @adamsilverstein <https://github.com/adamsilverstein> other than the one
> minor text change this is [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WP-API/WP-API/pull/1553#issuecomment-140588958>.
>



-- 
Adam Silverstein
Senior Web Engineer @ 10up

**Comment by adamsilverstein** [comment]
Corrected: JSON API -> WP REST API

**Comment by rachelbaker** [comment]
Merged #1553

**Pull request merged**